api/python/t4/snapshots.py

```diff
@@ -400,9 +404,22 @@ def update(self, logical_key, entry):
         Returns:
             A new package
         """
+        if 'meta' in kwargs and entry is None:
+            return self._update_meta(logical_key, kwargs['meta'])
+
+        pkg = self._clone()
+        if isinstance(entry, str):
```

Comment:
Update the docstring arg type to reflect that it can be a string now.

Reply:
done